gui-tk/minsky.tcl

```diff
@@ -953,7 +977,20 @@ proc variableContext {x y X Y} {
 		}
     }
     tk_popup .variables.context $X $Y
-}  
+}
+
+# Don't understand why this doesn't work???
```

Comment:
What happens currently?

Reply:
Nothing at all. I wonder if should create a tabs.tcl file and source it in minsky.tcl?